### ISSUE TITLE
[Bug]: Does not read .env file to get OpenAI key.

### ISSUE THREAD

**Issue by ChildOf7Sins** [Pythagora-io/gpt-pilot#731]
### Version

VisualStudio Code extension

### Operating System

Windows 11

### What happened?

When I launch Pythagora in VSCode it says that my free trial has expired, but I already have my OpenAI key in the .env file. The error message also says the wrong key and endpoint. I have revoked this API key already.

![image](https://github.com/Pythagora-io/gpt-pilot/assets/131901009/ab803789-dcbd-404d-94b6-08f189ee1cbf)

![image](https://github.com/Pythagora-io/gpt-pilot/assets/131901009/9f2f106e-bfc6-4446-a36c-5b06ff132349)

![image](https://github.com/Pythagora-io/gpt-pilot/assets/131901009/df5328f0-9ab6-467b-80c3-dd21310ae1b2)

**Comment by 80Builder80** [comment]
I was having the same issue. I found that my .env file was not in the correct location. It was located in the /gpt-pilot/ folder. Once I moved it to /gpt-pilot/pilot/ all was well.

**Comment by bfalkenstein** [comment]
I have the same issue. My .env file is located in the right folder (/gpt-pilot/pilot/). It's still using the wrong key (...b6c2 is not my key), and the wrong endpoint (api.trybricks.ai instead of api.openai.com)
![error](https://github.com/Pythagora-io/gpt-pilot/assets/14105595/c2f7a593-603b-4e9f-81da-c35e2ca63549)

**Comment by Alexawy82** [comment]
I am facing the same problem, the .env file is in the correct location as well. I have my chatgpt API there and it's not being touched I can't get out of that subscription screen

**Comment by rcalastro** [comment]
Same here
![image](https://github.com/Pythagora-io/gpt-pilot/assets/3905534/8dabd972-2eb6-47ce-8bbb-29d1d4077ba1)

**Comment by actus-1** [comment]
Same issue here

**Comment by nzkritik** [comment]
I have the same issue as above, and removing the environment variable and rebooting the system did not resolve it for me.
![image](https://github.com/Pythagora-io/gpt-pilot/assets/22699493/cfdd9bb2-0bca-4651-94eb-6f0ec3478821)
I'll try a few other options and poke around a bit

**Comment by ChildOf7Sins** [comment]
> also same issue here
> 
> **EDIT:** i think i found a solution for my case. i found out i had saved another OpenAI-API-Key in the **Environment Variables** in Windows 11.
> 
> After deleting this Variable and restarting VSCode everything seems working fine.

This could be it? Where did you look to find it? My Windows system's variables don't have and OpenAI API keys, only a google one.

**Comment by ChildOf7Sins** [comment]
I'm sorry everyone. My issue is fixed though I'm not entirely sure why and I think it's because GPT-Pilot and the VSCode plug in are different. I originally used git to pull down the gpt-pilot repository then pointed the plugin to that directory.

I decided to delete my gpt-pilot folder and start over. This time, I let the plugin handle pulling the repository and setting up it's workspace. Then I found the .env file and added my key to it. Works fine now and I confirmed it's using my OpenAI tokens.
